pubsub/redis/redis.go

```diff
@@ -89,20 +95,20 @@ func parseRedisMetadata(meta pubsub.Metadata) (metadata, error) {
 	}
 
 	if val, ok := meta.Properties[processingTimeout]; ok && val != "" {
-		if processingTimeoutMS, err := strconv.ParseUint(val, 10, 64); err == nil {
-			m.processingTimeout = time.Duration(processingTimeoutMS) * time.Millisecond
+		if processingTimeoutMs, err := strconv.ParseUint(val, 10, 64); err == nil {
+			m.processingTimeout = time.Duration(processingTimeoutMs) * time.Millisecond
 		} else if d, err := time.ParseDuration(val); err == nil {
 			m.processingTimeout = d
 		} else {
 			return m, fmt.Errorf("redis streams error: can't parse processingTimeout field: %s", err)
 		}
 	}
 
-	if val, ok := meta.Properties[reclaimInterval]; ok && val != "" {
-		if reclaimIntervalMS, err := strconv.ParseUint(val, 10, 64); err == nil {
-			m.reclaimInterval = time.Duration(reclaimIntervalMS) * time.Millisecond
+	if val, ok := meta.Properties[redeliverInterval]; ok && val != "" {
+		if redeliverIntervalMs, err := strconv.ParseUint(val, 10, 64); err == nil {
+			m.redeliverInterval = time.Duration(redeliverIntervalMs) * time.Millisecond
 		} else if d, err := time.ParseDuration(val); err == nil {
-			m.reclaimInterval = d
+			m.redeliverInterval = d
 		} else {
 			return m, fmt.Errorf("redis streams error: can't parse processingTimeout field: %s", err)
```

Comment:
```suggestion
			return m, fmt.Errorf("redis streams error: can't parse redeliverInterval field: %s", err)
```

redeliverInterval field